src/app/components/SearchFilter/SearchFilter.tsx

```diff
@@ -27,10 +28,10 @@ export const SearchFilter: FunctionComponent<FilterProps> = ({
 }) => {
   const { t } = useTranslation();
   const [valid, setValid] = useState<boolean>(true);
-  const [currentSearch, setCurrentSearch] = useState<string | undefined>();
+  const [currentSearch, setCurrentSearch] = useState<string>('');
   const inputRef = useRef<HTMLInputElement>(null);
 
-  const onChange = (currentSearch?: string) => {
+  const onChange = (currentSearch: string) => {
```

Comment:
Getting rid of the react console warning about the component changing an uncontrolled input to be controlled.